xml/repeated-content-decl.ent

```diff
@@ -0,0 +1,118 @@
+Repeated content entities, for re-printing content in multiple locations
+rather than using xrefs, so that users do not have to skip around so much
+This works just like copy-paste, so be mindful of sections, titles, and
+paragraphs
+............................................................................-->
+
+<!-- Use xmlns="http://docbook.org/ns/docbook" on the first occurrence of every
+formatting tag in each entity, to avoid "Namespace default prefix was not found" 
+errors -->
+
+<!ENTITY namespaces 
+'<formalpara xmlns="http://docbook.org/ns/docbook">
+<title>Length of release names</title> 
+<para xmlns="http://docbook.org/ns/docbook">
+Release names (e.g. when you run <command>helm install --name</command>) 
+have a maximum length of 36 characters.
+</para>
+</formalpara>
+<formalpara xmlns="http://docbook.org/ns/docbook">
+<title>Always install to a fresh namespace</title>
+<para xmlns="http://docbook.org/ns/docbook">
+    If you are not creating a fresh &productname; installation, but have deleted 
+    a previous deployment and are starting over, you must create new namespaces
+    with the 
+    <command>kubectl create namespace <replaceable>name</replaceable></command>
+        command. 
+    Do not re-use your old namespaces. The <command>helm delete</command> command 
+    does not remove generated secrets from the SCF and UAA namespaces as it is 
+    not aware of them. These leftover secrets will cause problems. See 
+    <xref linkend="sec.cap.rebuild.depl"/> for more information.
+    </para>
+    </formalpara>'>
+    
+<!ENTITY readmefirst
+  '<important xmlns="http://docbook.org/ns/docbook">
+      <title xmlns="http://docbook.org/ns/docbook">README first!</title>
+      <para xmlns="http://docbook.org/ns/docbook">
+       Before you start deploying &productname;, review the following documents:
+      </para>
+      <itemizedlist xmlns="http://docbook.org/ns/docbook">
+       <listitem>
+        <para>
+         Release Notes:
+         <link xmlns:xlink="http://www.w3.org/1999/xlink" 
+         xlink:href="https://www.suse.com/releasenotes/x86_64/SUSE-CAP/1/"/>
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         <xref linkend="cha.cap.depl.notes"/>
+        </para>
+       </listitem>
+      </itemizedlist>
+    </important>'>    
+
+<!ENTITY dns-management
+'<para xmlns="http://docbook.org/ns/docbook">
+<!--TODO link to DNS requirements section, when it exists -->
+You must have control of your own DNS management, and set up all the necessary 
+domains and subdomains for &productname; and your applications.
+</para>'>
+    
+<!ENTITY releases-table    
+'<!-- TODO remember to keep this table updated -->
+    <informaltable xmlns="http://docbook.org/ns/docbook" >
+   <tgroup cols="3">
+    <thead xmlns="http://docbook.org/ns/docbook">
+     <row xmlns="http://docbook.org/ns/docbook">
+      <entry xmlns="http://docbook.org/ns/docbook" >CAP Release</entry>
```

Comment:
Same here, you don't need the DocBook namespaces. Remove them from `entry`, `row` and `thead`.